packages/types/README.md

```diff
@@ -1,10 +1,55 @@
-0x types
+@0xproject/types
 ------
 
-TS types shared across 0x projects and packages
+Typescript types shared across 0x projects and packages
 
-## Install
+## Installation
 
 ```bash
 yarn add -D @0xproject/types
 ```
+
+## Usage
+
+```javascript
+import {
+    TransactionReceipt, 
+    TxData,
+    TxDataPayable,
+} from '@0xproject/types';
+```
+
+
+## Contributing
+
+We strongly encourage our community members to help us make improvements and to determine the future direction of the protocol. To report bugs within this package, please create an issue in this repository. 
```

Comment:
Can we change this to `We strongly recommend the community to help us...` everywhere? I updated the template.